app/response/views.py

```diff
@@ -78,9 +78,21 @@ def response_note(request_id):
     """
     note_data = flask_request.form
 
-    required_fields = ['content',
-                       'email-note-summary',
-                       'privacy']
+    current_request = Requests.query.filter_by(id=request_id).first()
+    required_fields = []
+    privacy = None
+    is_editable = True
+    is_requester = False
+
+    if current_user.is_agency:
+        required_fields.extend(['content',
+                                'email-note-summary',
+                                'privacy'])
+    else:
```

Comment:
Should this be `elif current_user == current_request.requester` and have an else that raises a custom exception?